### PR TITLE
player/video: mention --profile=fast and --hwdec in AV desync message

### PR DESCRIPTION
**Pull request by kasper93**

### CODE REVIEW

File: player/video.c [hunk 1]
```diff
@@ -59,6 +59,7 @@ static const char av_desync_help_text[] =
 "Audio/Video desynchronisation detected! Possible reasons include too slow\n"
 "hardware, temporary CPU spikes, broken drivers, and broken files. Audio\n"
 "position will not match to the video (see A-V status field).\n"
+"Consider trying `--profile=fast` and/or `--hwdec=auto-safe`, as they may help.\n"
```
Comment (Dudemanguy) [inline player/video.c:62]: Totally useless grammar nit but I think (?) you don't need the comma.

Comment (kasper93) [reply player/video.c]: I have no idea. Seems redundant indeed, removed.

**Review by Dudemanguy — APPROVED** [review]

File: player/video.c [hunk 2]
```diff
@@ -59,6 +59,7 @@ static const char av_desync_help_text[] =
 "Audio/Video desynchronisation detected! Possible reasons include too slow\n"
 "hardware, temporary CPU spikes, broken drivers, and broken files. Audio\n"
 "position will not match to the video (see A-V status field).\n"
+"Consider trying `--profile=fast` and/or `--hwdec=auto-safe` as they may help.\n"
```
Comment (N-R-K) [inline player/video.c:62]: Maybe get rid of the `--` as these are better set in `mpv.conf` instead of setting them in the cli everytime.

**Review by N-R-K — CHANGES_REQUESTED** [review]

File: player/video.c (hunk 2 above)
Comment (kasper93) [reply player/video.c:62]: We refer to option with `--` in all documents, so it seems consistent to keep it here too.

Comment (N-R-K) [reply player/video.c:62]: For reference, this is the warning you get when using `vdpau`

image

(notice it says `vo=gpu` and not `--vo=gpu` etc).

Comment (guidocella) [reply player/video.c:62]: Note that `--profile=fast` still works in `mpv.conf` anyway.

Comment (kasper93) [reply player/video.c:62]: Seems to be one outlier. Few lines above you have with `--` https://github.com/mpv-player/mpv/blob/ef11d31c3acfd71307a94e44ad164a4861287675/video/out/vo_vdpau.c#L1039-L1042

I think `--` clearly denotes that we refer to the option and anyone who modified config knows what it means.

**Review by haasn — APPROVED** [review]

**Pull request merged**